torch/csrc/jit/mobile/model_bytecode_version.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <istream>
+#include <memory>
+
+namespace caffe2 {
+namespace serialize {
+class ReadAdapterInterface;
+} // namespace serialize
+} // namespace caffe2
+
+namespace torch {
+namespace jit {
+namespace mobile {
+
+// The family of methods below to get bytecode version from a model
```

Comment:
Does bytecode exist outside of the mobile space?

Reply:
For serialization APIs (both export and import) we don't put them in mobile namespace. 
1. Those APIs are also used in server side. For example, at deployment stage.
2. torch::jit::_load_for_mobile, the name may already differentiates from torch::jit::load. Adding another namespace is not necessary.

Reply:
Theres nothing about the name of this method that screams mobile to me though, and the the model ops stuff will have the same problem but worse. I'm not sure I see the harm in essentially labeling that hey this stuff is mobile specific it wont work unless you pass in the file path of a mobile module.

Reply:
@JacobSzwejbka Either function name or file path could give a hint.

Reply:
1. Namespace: I think it's good to remove mobile in this PR as discussed above.
2. File organization: it can be done in a separate PR later (tracking in T89985123) 
* The serialization related codes (`import.h/cpp`, export_data.cpp`, compatibility.h/cpp`, etc) should be moved to torch/csrc/jit/serialization/mobile. 
* torch/csrc/jit/mobile should be renamed to torch/csrc/jit/mobile_runtime to make this folder runtime specific.